src/main/webapp/WEB-INF/classes/log4j2.properties

```diff
@@ -0,0 +1,23 @@
+rootLogger.level = INFO
```

Comment:
Same as in the other review, did you manually check more messages are being logged when switching log level?

Reply:
yes tested see my comment below.